iml-mailbox/src/main.rs

```diff
@@ -11,16 +11,23 @@
 
 use futures::{Stream, StreamExt};
 use iml_mailbox::ingest_data;
+use iml_manager_env::get_pool_limit;
 use iml_postgres::{get_db_pool, sqlx::PgPool};
 use iml_tracing::tracing;
+use lazy_static::lazy_static;
 use std::pin::Pin;
 use warp::Filter as _;
 
+// Default pool limit if not overridden by POOL_LIMIT
+lazy_static! {
```

Comment:
Why lazy_static?

Reply:
So it's only eval'd once, and this allows it to be used inside the `async move` without passing in as an extra filter.